pages/index.js

```diff
@@ -155,7 +155,7 @@ class IndexPage extends Component {
           </section>
           <section className='h-section pt0-ns pt4-l'>
             <div className='flex flex-column pt4-ns'>
-              <h2 className='f2-l f3 tf-oswald tc mb5-l mv4'>
+              <h2 className='f2-l f3 tf-oswald tc mb5-l mv4' style={{fontSize:80,fontWeight:"bold"}}>
```

Comment:
the font size 40 for the other headers look good, not sure why this is font size 80 and bold